app/models/freelancer.rb

```diff
@@ -1,5 +1,5 @@
 class Freelancer < ActiveRecord::Base
-  belongs_to :user, dependent: :destroy
+  belongs_to :user
```

Comment:
That was Cyrille helping me debug the problem with the client freelancer creation, it was messing with the code